### PR TITLE
[7.3.x] JBPM-6103 Stunner - Script Language values are not persist to Embedded subprocess

### PR DESCRIPTION
**Pull request by tiagodolphine**
Port back commit feb504a6d5f2344c57edf8b47180cc832f47b27c to the branch 7.3.x.
Related PR https://github.com/kiegroup/kie-wb-common/pull/1070

@manstis 
@hasys

### CODE REVIEW

**Comment by manstis** [comment]
@hasys Would you like a look? This is a back-port of a PR approved by @romartin and merged.

**Comment by hasys** [comment]
Jenkins execute full downstream build

File: kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-api/src/main/java/org/kie/workbench/common/stunner/bpmn/definition/EmbeddedSubprocess.java [hunk 1]
```diff
@@ -154,8 +158,7 @@ public EmbeddedSubprocess(final @MapsTo("general") BPMNGeneralSet general,
         this.onEntryAction = onEntryAction;
         this.onExitAction = onExitAction;
         this.scriptLanguage = scriptLanguage;
-        this.isAsync = isAsync;
-        this.processData = processData;
+        this.isAsync = isAsync;this.processData = processData;
```
Comment (hasys) [inline kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-api/src/main/java/org/kie/workbench/common/stunner/bpmn/definition/EmbeddedSubprocess.java]: Not needed to do this :)

**Comment by romartin** [comment]
Hey @tiagodolphine , please when possible just fix the formatting stuff and let's try to get this merged and close pull requests... :this is an easy one! :) Thanks!!

**Comment by tiagodolphine** [comment]
@romartin formatting is fixed.

**Comment by manstis** [comment]
@hasys Waiting on your approval.

**Pull request merged**